### DISCUSSION TITLE
[Scam Alert] Emway باشگاه مشتریان ام وی

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#38] [Announcements]
<div dir="rtl">

**Project Name:** 
Emway

**نام پروژه:** 
باشگاه مشتریان ام وی

**نماد پروژه:**

**اتفاقی که افتاده است:**
یه شرکت هست به این ادرس 

http://emway.ir

شرکت که چه عرض کنم، تو خود سایتش زده فروشگاه اینترنتی

نحوه کارش هم اینطوریه که شما 2 میلیون میدید عضو میشید. بعد هر نفری هم که بیارید باید 2 میلیون بده و فکر کنم 200 تومن هم به سرشاخه میرسه.

این روند تا ده سطح ادامه داره.

البته اینم بگم که کنارش به شما یه کارت تخفیف باشگاه مشتریان هم میده که میتونید از فروشگاه های طرف قرارداد خرید کنید و تخفیف بگیرید.

که خب این موردش فقط برای رد گم کنی هست و 99 درصد افراد برای اون سودی که از زیرمجموعه گیری می گیرند عضوش میشند.

تو این سایت توضیحات کاملی رو در مورد این کلاهبرداری ارائه

https://miladhabibollahi.com/emway/

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار ریسک بسیار بالا

**نوع کلاهبرداری:** 
طرح پانزی، طرح هرمی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
http://emway.ir

**برآورد مبلغ کلاهبرداری:**

**منابع:**
http://www.iranhoshdar.ir/Newsb/1/41701
https://miladhabibollahi.com/emway/

**جمع بندی:**

**Comment by saeedkousha** [comment]
<html>
<body>


اختصاصی ایران هشدار- پیرو سؤالات متداول کاربران گرامی در خصوص فعالیت شبکه­ ای شرکت emway قابل ذکر است، این شرکت فاقد پروانه کسب بازاریابی شبکه ­ای بوده و در صورت مبادرت به فعالیت شبکه­ ای در قالب مجموعه­ سازی و پرداخت پورسانت به صورت چند لایه فعالیت آن غیرقانونی تلقی می­ گردد.البته اخیراً شرکت مذکور به منظور سرپوش گذاشتن بر فعالیت غیرقانونی خود اقدام به ادغام با یکی از شرکت­ های دارای پروانه کسب نموده و تبلیغات گسترده ­ای را در جهت مشروعیت بخشی فعالیت خود به راه انداخته است.در این راستا قابل ذکر است در صورت ادغام هریک از شرکت­ های فاقد مجوز با شرکت­ های بازاریابی شبکه­ ای، فقط طرح و کالاهای دارای مجوز فروش شرکت دارای پروانه کسب، قابلیت عرضه در این سیستم را دارد، اما شرکت غیرقانونی ام وی اقدام به ارائه طرح درآمدی و محصولات خود در پوشش نام شرکت دارای پروانه کسب نموده که مصداق فعالیت غیرقانونی می­ باشد.
--
http://www.iranhoshdar.ir/Newsb/1/41701


</body>
</html>
